website/style.css

```diff
@@ -1,421 +1,442 @@
+  position: fixed;
+  top: 0;
+  display: flex;
+  flex-direction: row;
+  margin: auto;
+  align-items: stretch;
+  height: 10vh;
+  z-index: 4;
+  width: 100%;
 }
 
 .header-text {
-    display: flex;
-    flex-direction: column;
-    background-color: black;
-    justify-content: center;
-    font-size: 20px;
+  display: flex;
+  flex-direction: column;
+  background-color: black;
+  justify-content: center;
+  font-size: 20px;
 }
 
 .header-text-title {
-    flex: 3;
-    background-color: #181818;
-    font-weight: 400;
+  flex: 3;
+  background-color: #181818;
+  font-weight: 400;
 }
 
 .header-text-title h1 {
-    margin-left: 30px;
+  margin-left: 30px;
 }
 
 .header-text-contact {
-    flex: 1;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    background-color: #232323;
-    text-decoration: none;
-    animation: flexAnimation 1s;
-}
-
-.flex-all-content {
-    display: flex;
-    flex-direction: row;
-    margin: auto;
-    flex-wrap: nowrap;
-    justify-content: space-evenly;
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #232323;
+  text-decoration: none;
+  animation: flexAnimation 1s;
 }
 
-.line {
-    display: flex;
-    justify-content: space-evenly;
+.content {
+  display: flex;
+  flex-direction: row;
+  margin: auto;
+  flex-wrap: nowrap;
+  justify-content: space-evenly;
 }
 
-.line div {
-    border-right: red 5px dashed;
-    animation: fadeInBottom 2s;
+.line {
+  display: flex;
+  justify-content: space-evenly;
 }
 
-.gradient-contatiner {
-    position: fixed;
-    top: 0;
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-    width: 100%;
-    height: 100vh;
+.line div {
+  border-right: red 5px dashed;
+  animation: fadeInBottom 2s;
 }
 
-.gradient-push {
-    flex: auto;
-    width: 909px;
+.main-content {
+  z-index: 2;
 }
 
-.gradient {
-    flex: auto;
-    background-image: linear-gradient(to top, black, black, rgba(255,0,0,0), rgba(255,0,0,0), black, black)
+.gradient-for-line {
+  position: fixed;
+  z-index: 2;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-image: linear-gradient(
+    to top,
+    black,
+    black,
+    rgba(255, 0, 0, 0),
+    rgba(255, 0, 0, 0),
+    black,
+    black
+  );
 }
 
-p span {
-    font-size: 30px;
-    line-height: 40px;
+h3 {
+  font-size: 30px;
+  line-height: 40px;
 }
 
 .text-image-flex {
-    display: flex;
-    flex-direction: row;
-    justify-content: start;
-    animation: fadeInRight 2s;
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  animation: fadeInRight 2s;
 }
 
-.image-container-3 {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    margin: auto;
+.portrait-img {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: auto;
 }
 
-.image-container-3 img {
-    height: 300px;
+.portrait-img img {
```

Comment:
Suggestion: Get rid of non class selectors or general selectors or child general selectors with direct class selectors

Reply:
Not really. Found these for u
```
.header-text-title h1
.portrait-img img 
.different-mediums-img img
.languages-and-frameworks-img img
.header-text h1
.header-text h2
header h1
.header-text-contact h2
```

Reply:
a1d6127

Reply:
48aa222

Forgot to rebase this. I am going to learn this now tho (sorry for the inconvenience(i am aware that this is not acceptable in a professional environment but since this is less of an important project and I ran out of time (had to go do something) I believe it is wise to think that you will let this one slide))